tests/install_target_test.go

```diff
@@ -44,7 +45,19 @@ var _ = Describe("kairos install test different targets", Label("install-test-ta
 	})
 
 	AfterEach(func() {
+		if CurrentSpecReport().Failed() {
+			serial, _ := os.ReadFile(filepath.Join(vm.StateDir, "serial.log"))
+			_ = os.MkdirAll("logs", os.ModePerm|os.ModeDir)
+			_ = os.WriteFile(filepath.Join("logs", "serial.log"), serial, os.ModePerm)
```

Comment:
do we need to `.gitignore` this?

Reply:
we never did anywhere I think but it will be good for local testing?